soc/riscv/riscv-ite/common/soc_dt.h

```diff
@@ -41,4 +41,123 @@
 		     inst)					\
 	}
 
+/*
+ * For it8xxx2 wake-up controller (WUC)
+ */
+#define IT8XXX2_DEV_WUC(idx, inst)					\
+	DEVICE_DT_GET(DT_PHANDLE(IT8XXX2_DT_INST_WUCCTRL_0(inst, idx), wucs))
+#define IT8XXX2_DEV_WUC_GROUP(idx, inst)				\
+	DT_PHA(IT8XXX2_DT_INST_WUCCTRL_0(inst, idx), wucs, group)
+#define IT8XXX2_DEV_WUC_MASK(idx, inst)					\
+	DT_PHA(IT8XXX2_DT_INST_WUCCTRL_0(inst, idx), wucs, mask)
+
+/**
+ * @brief For it8xxx2, get a node identifier for a phandle in a wucctrl
```

Comment:
Can you provide an example for how the `wucctrl-n` properties are used?

Reply:
In middle of this comment has two example for wucctrl-0 and wucctrl-1:  
IT8XXX2_DT_WUCCTRL_BY_IDX(DT_NODELABEL(n), 0, 1) // DT_NODELABEL(bar)
IT8XXX2_DT_WUCCTRL_BY_IDX(DT_NODELABEL(n), 1, 0) // DT_NODELABEL(baz)
So shall I add more example?

Reply:
This [draft CL](https://crrev.com/c/3359143) is a good example showing how to use the `wucctrl` property.  Do you plan to create a PR for that change?

Reply:
Yes, I will create a GPIO PR with WUC changes.